Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -286,20 +286,17 @@ def populate(self, issue, member=None):
             issue._references = []
             refs = set()
 
-            # Attempt to match radar importer first
-            if self.radar_importer:
-                for comment in (issue.comments or []):
-                    if not comment:
-                        continue
-                    if comment.user != self.radar_importer:
-                        continue
-                    candidate = GenericTracker.from_string(comment.content)
-                    if not candidate or candidate.link in refs or (isinstance(type(candidate.tracker), type(issue.tracker)) and candidate.id == issue.id):
-                        continue
-                    issue._references.append(candidate)
-                    refs.add(candidate.link)
+            # In reverse chronological order, prioritize comments with only a bug url, then all comments, then description
+            for comment in reversed(issue.comments or []):
+                if not comment:
+                    continue
+                candidate = GenericTracker.from_string(comment.content)
+                if not candidate or candidate.link in refs or (isinstance(type(candidate.tracker), type(issue.tracker)) and candidate.id == issue.id):
+                    continue
+                issue._references.append(candidate)
+                refs.add(candidate.link)
 
-            for text in [issue.description] + [comment.content for comment in (issue.comments or []) if comment]:
+            for text in reversed([issue.description] + [comment.content for comment in (issue.comments or []) if comment]):
```

Comment:
I'd suggest going with chained iterators, and use iterators throughout. From memory, fetching comment contents has a cost for some issue trackers.

Thus, something like:

```python
from itertools import chain

for text in chain(
    (comment.content for comment in (reversed(issue.comments) or ()) if comment),
    (issue.description,),
):
    pass
```

(This also uses tuples and not lists, which is really meaningless optimisation, but it doesn't hurt.)